diff --git a/Testcases1/msmonitor/test_monitor_cc.sh b/Testcases1/msmonitor/test_monitor_cc.sh
index 005b2c36f..2c46fa833 100644
--- a/Testcases1/msmonitor/test_monitor_cc.sh
+++ b/Testcases1/msmonitor/test_monitor_cc.sh
@@ -26,7 +26,7 @@ main() {
     bash examples/llama2/test_cc.sh  > ${testcase_result_dir}/plog.txt 2>&1
     
     verify_csv_output 1
-    expected_header="param_name,step,norm,min,max,mean,nans"
+    expected_header="vpp_stage,name,step,norm,min,max,mean,nans"
     # to change after pr 2392
     # expected_header="cc_name,post_0.norm,post_0.min,post_0.max,post_0.mean,post_0.nans,post_1.norm,post_1.min,post_1.max,post_1.mean,post_1.nans"
     verify_csv_content ${testcase_result_dir}/*/other_0-0.csv 6 $expected_header
diff --git a/Testcases1/msmonitor/test_monitor_mv.sh b/Testcases1/msmonitor/test_monitor_mv.sh
index 77fd8fa82..6c23b75e4 100644
--- a/Testcases1/msmonitor/test_monitor_mv.sh
+++ b/Testcases1/msmonitor/test_monitor_mv.sh
@@ -28,10 +28,10 @@ main() {
     verify_csv_output 2
 
     # testcase_result_dir=/home/maoyanlong/att_smoke_new/WorkSpace1/output-16/msmonitor/test_mv
-    expected_header="vpp_stage,param_name,step,norm,min,max,mean,nans"
+    expected_header="vpp_stage,name,step,norm,min,max,mean,nans"
     # expected_header="vpp_stage,param_name,step,exp_avg.norm,exp_avg.min,exp_avg.max,exp_avg.mean,exp_avg.nans"
     verify_csv_content ${testcase_result_dir}/*/exp_avg_1-1.csv 28 $expected_header
-    expected_header="vpp_stage,param_name,step,norm,min,max,mean,nans"
+    expected_header="vpp_stage,name,step,norm,min,max,mean,nans"
     # expected_header="vpp_stage,param_name,step,exp_avg_sq.norm,exp_avg_sq.min,exp_avg_sq.max,exp_avg_sq.mean,exp_avg_sq.nans"
     verify_csv_content ${testcase_result_dir}/*/exp_avg_sq_1-1.csv 28 $expected_header
 
diff --git a/Testcases1/msmonitor/test_monitor_wg.sh b/Testcases1/msmonitor/test_monitor_wg.sh
index 8355bf853..542c502d4 100644
--- a/Testcases1/msmonitor/test_monitor_wg.sh
+++ b/Testcases1/msmonitor/test_monitor_wg.sh
@@ -30,7 +30,7 @@ main() {
     verify_csv_output 2
 
     # testcase_result_dir=/home/maoyanlong/att_smoke_new/WorkSpace1/output-16/msmonitor/test_wg
-    expected_header="vpp_stage,param_name,step,norm,min,max,mean,nans"
+    expected_header="vpp_stage,name,step,norm,min,max,mean,nans"
     verify_csv_content ${testcase_result_dir}/*/grad_reduced_0-0.csv 28 $expected_header
     verify_csv_content ${testcase_result_dir}/*/grad_unreduced_0-0.csv 28 $expected_header
 
diff --git a/Testcases1/msmonitor/test_monitor_xy_forward.sh b/Testcases1/msmonitor/test_monitor_xy_forward.sh
index d0bd96a53..3d06ab03f 100644
--- a/Testcases1/msmonitor/test_monitor_xy_forward.sh
+++ b/Testcases1/msmonitor/test_monitor_xy_forward.sh
@@ -29,8 +29,8 @@ main() {
     verify_csv_output 1
 
     # testcase_result_dir=/home/maoyanlong/att_smoke_new/WorkSpace1/output-16/msmonitor/test_xy
-    expected_header="vpp_stage,module_name,step,input.norm,input.min,input.max,input.mean,input.nans,output.norm,output.min,output.max,output.mean,output.nans"
-    verify_csv_content ${testcase_result_dir}/*/actv_0-0.csv 1201 $expected_header
+    expected_header="vpp_stage,name,step,micro_step,norm,min,max,mean,nans"
+    verify_csv_content ${testcase_result_dir}/*/actv_0-0.csv 2401 $expected_header
 
     end_time=$(date "+%s")
     duration_time=$(( ${end_time} - ${start_time} ))
diff --git a/Testcases1/msmonitor/test_monitor_xy_forward_single_ops.sh b/Testcases1/msmonitor/test_monitor_xy_forward_single_ops.sh
index 01219843c..7a41e8550 100644
--- a/Testcases1/msmonitor/test_monitor_xy_forward_single_ops.sh
+++ b/Testcases1/msmonitor/test_monitor_xy_forward_single_ops.sh
@@ -32,8 +32,8 @@ main() {
 
     # testcase_result_dir=/home/maoyanlong/att_smoke_new/WorkSpace1/output-16/msmonitor/test_xy
     
-    expected_header="vpp_stage,module_name,step,input.norm,output.norm"
-    verify_csv_content ${testcase_result_dir}/*/actv_0-0.csv 1201 $expected_header
+    expected_header="vpp_stage,name,step,micro_step,norm"
+    verify_csv_content ${testcase_result_dir}/*/actv_0-0.csv 2401 $expected_header
 
     end_time=$(date "+%s")
     duration_time=$(( ${end_time} - ${start_time} ))
